db/people.yml

```diff
@@ -2040,3 +2040,263 @@ zzgong:
+  employment:
+    - begin_year: 2008
+      position: Assistant Professor
+      organization: University of Jordan
+    - begin_year: 2004
+      position: Research Associate
+      advisor: sbillinge
+      organization: msu
+      status: phd
+      end_year: 2007
+      end_month: 7
+      group: bg
+  name: Moneeb Shatnawi
+  position: Assistant Professor
+hyavas:
+  active: false
+  aka:
+    - Hasan Yavas
+  avatar: https://static-exp1.licdn.com/sc/h/djzv59yelk5urv2ujlazfyvrk
+  bio: bio
+  education:
+    - begin_year: 2001
+      degree: PhD
+      institution: msu
+      end_year: 2007
+      end_month: 10
+      group: bg
+      advisor: sbillinge
+  employment:
+    - begin_year: 2018
+      position: Staff Scientist
+      organization: SLAC National Accelerator Laboratory
+    - begin_year: 2001
+      position: Research Associate
+      advisor: sbillinge
+      organization: msu
+      status: phd
+      end_year: 2007
+      end_month: 10
+      group: bg
+  name: Hasan Yavas
+  position: Staff Scientist
+tdavis:
+  active: false
+  aka:
+    - Timur Dykhne
+    - Timur Davis
+  avatar: https://static-exp1.licdn.com/sc/h/djzv59yelk5urv2ujlazfyvrk
+  bio: bio
+  education:
+    - begin_year: 2009
+      degree: PhD
+      institution: columbiau
+      end_year: 2011
+      end_month: 6
+      group: bg
+      advisor: sbillinge
+  employment:
+    - begin_year: 2019
+      position: Investment lead
+      organization: Munich Re Venture
+    - begin_year: 2013
+      position: Investment Manager
+      organization: Samsung Venture Investment
+      end_year: 2018
+    - begin_year: 2009
+      position: Research Associate
+      advisor: sbillinge
+      organization: columbiau
+      status: phd
+      end_year: 2011
+      end_month: 6
+      group: bg
+  name: Timur Davis
+  position: Investment lead
+amasadeh:
+  active: false
+  aka:
+    - Ahmad Masadeh
+  avatar: https://static-exp1.licdn.com/sc/h/djzv59yelk5urv2ujlazfyvrk
+  bio: bio
+  education:
+    - begin_year: 2003
+      degree: PhD
+      institution: msu
+      end_year: 2008
+      end_month: 1
+      group: bg
+      advisor: sbillinge
+  employment:
+    - begin_year: 2008
+      position: Assistant Professor
+      organization: University of Jordan
+    - begin_year: 2003
+      position: Research Associate
+      advisor: sbillinge
+      organization: msu
+      status: phd
+      end_year: 2008
+      end_month: 6
+      group: bg
+  name: Ahmad Masadeh
+  position: Assistant Professor
+hlin:
+  active: false
+  aka:
+    - He lin
+  avatar: https://static-exp1.licdn.com/sc/h/djzv59yelk5urv2ujlazfyvrk
+  bio: bio
+  education:
+    - begin_year: 2000
+      degree: PhD
+      institution: msu
+      end_year: 2006
+      end_month: 12
+      group: bg
+      advisor: sbillinge
+  employment:
+    - begin_year: 2009
+      begin_month: 12
+      position: Staff Scientist
+      organization: Shangai Synchrotron Radiation Facility
```

Comment:
typo.   I think if you use ``ssrf`` it will dereference properly from institutions, i.e., things will build with the right name.